services/galley/src/Galley/API/Create.hs

```diff
@@ -647,8 +647,8 @@ toUUIDs a b = do
   b' <- U.fromUUID (toUUID b) & note InvalidUUID4
   return (a', b')
 
-accessRole :: NewConv -> AccessRole
-accessRole b = fromMaybe Data.defRole (newConvAccessRole b)
+accessRoles :: NewConv -> Set AccessRoleV2
+accessRoles b = fromMaybe Data.defRole (newConvAccessRoles b)
```

Comment:
If you follow [these suggestions](https://github.com/wireapp/wire-server/pull/2035#discussion_r786602136), you may want to expose `defRole` after all for this.  My mistake, sorry.